web/crux-ui/src/pages/deployments.tsx

```diff
@@ -77,25 +76,27 @@ const DeploymentsPage = (props: DeploymentsPageProps) => {
   const defaultHeaderClass = 'h-11 uppercase text-bright text-sm bg-medium-eased py-3 pl-4 font-semibold'
   const headerClasses = [
     clsx('rounded-tl-lg', defaultHeaderClass),
-    ...Array.from({ length: headers.length - 2 }).map(() => defaultHeaderClass),
+    ...Array.from({ length: headers.length - 3 }).map(() => defaultHeaderClass),
     clsx('text-center', defaultHeaderClass),
-    clsx('rounded-tr-lg', defaultHeaderClass),
+    clsx('text-center rounded-tr-lg', defaultHeaderClass),
   ]
 
+  const onCellClick = (data: Deployment, row: number, col: number) => {
+    if (col >= headers.length - 1) {
```

Comment:
Better method to handle row clicks while handling action column clicks as well?

Reply:
Currently if the user clicks on the row in the actions column that will do nothing, so an action mislick won't navigate.
What do you mean by splitting the second section?

Reply:
Okay, never mind - I just misunderstood the question, as I can see now.